block/test/cardano-sl-block-test.cabal

```diff
@@ -0,0 +1,83 @@
+name:                cardano-sl-block-test
+version:             1.1.0
+synopsis:            Cardano SL - block processing (tests)
+description:         QuickCheck Arbitrary instances for Cardano SL block
+                     processing.
+license:             MIT
+license-file:        LICENSE
+author:              Serokell
```

Comment:
Are we supposed to be changing this?

Reply:
Yeah, although I haven't managed to get a new maintainer email from anyone!

For now put:
```
author: IOHK
maintainer: IOHK <support@iohk.io>
```

Reply:
With whitespace of course ;)